src/tim/vx/operation_private.h

```diff
@@ -45,13 +47,22 @@ class OperationImpl {
 
   vsi_nn_node_t* node() { return this->node_; }
 
+  std::vector<std::shared_ptr<Tensor>> InputsTensor() { return inputs_tensor_; }
+  std::vector<std::shared_ptr<Tensor>> OutputsTensor() {
+    return outputs_tensor_;
+  }
+
   GraphImpl* graph_;
   uint32_t operation_id_{0};
+  DataLayout layout_{DataLayout::WHCN};
```

Comment:
Add define DataLayout::ANY for the operator doesn't care about data layout, then we don't have to add new constructor for data layout. Just use DataLayout::ANY as the default value.

Reply:
ok